observability/prometheus/service-monitor.yaml

```diff
@@ -0,0 +1,16 @@
+templates:
+- |
+  apiVersion: monitoring.coreos.com/v1
+  kind: ServiceMonitor
+  metadata:
+    name: orchestrate
+    labels:
+      team: pegasys
+  spec:
+    selector:
+      matchLabels:
+        app.metrics: orchestrate
+    namespaceSelector:
+      any: true
```

Comment:
IMO it would be good to speficy the list of namespaces we are scrapping from to optimize the process